lib/resources/r4/medication_request.yaml

```diff
@@ -48,26 +79,88 @@ fields:
   required: 'No'
   type: code
   description: Medication Request Priority
+  action: get
   binding:
     description: Medication Request Priority
     terminology:
     - display: Request Priority
       system: http://hl7.org/fhir/request-priority
       info_link: https://hl7.org/fhir/r4/valueset-request-priority.html
+- name: doNotPerform
```

Comment:
What does Millennium do with this field?

Reply:
According to that wiki, we also don't support the field (it's greyed out). If that's true, why are we documenting it?

Reply:
Looking through the code, it seems like we do support it. I think the wiki probably just isn't up to date? Evan had to update it while he was removing the JBuilders, so I think that's the case for at least a few resources.

Reply:
it seems to me that this field is validated but unused, we can loop back on how that became the case. however, since it is validated, i think we should keep this note

Reply:
millennium doesn't use this field but since it is a modifier we verify that it cannot be true